### DISCUSSION TITLE
See if hours are invoiced or not

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/kimai#4738] [General] [answered]
Hello Kimai Community,
is it possible to filter tracked time that is billable and was already invoiced?

I do not see such a filter in the hours overview.

THX and best wishes

Niels

**Comment by kevinpapst** [comment]
Kimai has a "exported" flag, which is set both by the export module and the invoicing module.

Once you create an invoice, the exported flag is set to `true` for all included timesheets.

**Comment by niels-przybilla** [comment]
Hello @kevinpapst ,

thank you very much. But as I understand, there is no possibility to see this in frontend. Only via API?

THX and best wishes

Niels

**Reply by kevinpapst** [reply, marked as answer]
![image](https://github.com/kimai/kimai/assets/533162/c3beff0b-8c66-4651-9481-47217ccc6c74)

**Comment by niels-przybilla** [comment]
Thank you @kevinpapst .. let the blind see. Sorry for bothering you.
